### PR TITLE
[DOCS] Update scrolling document.asciidoc

### PR DESCRIPTION
**Pull request by salihklc**
adding more detail to example code

### CODE REVIEW

File: docs/search/scrolling-documents.asciidoc [hunk 1]
```diff
@@ -38,12 +38,13 @@ var searchResponse = Client.Search<Project>(s => s
...
 {
     ProcessResponse(searchResponse); <3>
-    searchResponse = Client.Scroll<Project>("10s", searchResponse.ScrollId);
+    searchResponse = Client.Scroll<Project>("10s", searchResponse.ScrollId); <4>
 }
 ----
 <1> Specify a scroll time for how long Elasticsearch should keep this scroll open on the server side. The time specified should be sufficient to process the response on the client side.
 <2> make subsequent requests to the scroll API to keep fetching documents, whilst documents are returned
-<3> do something with the response
+<3> do something with the response, (Client search request also return the first iteration of query)
```
Comment (stevejgordon) [inline docs/search/scrolling-documents.asciidoc:46]: I'd recommend we make this:
```
<3> Do something with the response (which includes the first "page" of results).
```

**Review by stevejgordon — CHANGES_REQUESTED** [review]
Thanks @salihklc for the suggestion. Unfortunately the docs process is quite complex right now. We're reworking the docs for v8 once that client is closer to GA. This change is best targeted at the 7.17 branch please.

Additionally, we generate the AsciiDoc file from [here](https://github.com/elastic/elasticsearch-net/tree/master/src/Tests/Tests/Search/ScrollingDocuments.doc.cs), so this change would need to be made in that source file and then the docs generator can be run from the command line using `.\build.bat documentation` which produces the asciidoc.

Is that something you're willing to take on? If so, once generated I can port the changes (if applicable) to the v8 documentation as part of that process.

If you're unable to do this locally, that's also fine and I can tackle it locally and close out this PR. Just let me know.

File: docs/search/scrolling-documents.asciidoc [hunk 2]
```diff
@@ -38,12 +38,13 @@ var searchResponse = Client.Search<Project>(s => s
...
+<4> Get next page scroll response and set it to searchReponse to use it on ProcessResponse
```
[the remark this answers is not in the document]
Comment (salihklc) [reply docs/search/scrolling-documents.asciidoc:47]: 🥳

**Comment by stevejgordon** [comment]
Thanks for your contribution, @salihklc. I will close this PR as we're reworking the 8.0 documentation significantly.

**Pull request closed without merging**